content/post/invitation-to-test.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Invitation to test : IR hosting service"
+date: 2020-09-28T17:00:00+02:00
+draft: false
+tags: ["iraas", "repository", "shared portal", "dedicated portal"]
+---
+
+One of the goals of the [SONAR project](https://sonar.ch/) is to set up a hosting service for institutional repositories (IRaaS - Institutional Repository as a Service). [Two variants](https://sonar.ch/iraas/) of this service are proposed: a shared portal and dedicated portals. In anticipation of its first production launch, this service is currently providing a public test instance at the address [sonar.test.rero.ch](https://sonar.test.rero.ch).
+
+<!--more-->
+
+In order to respond to the interest that it has been attracting in our community, the SONAR team is inviting all interested institutions to test this service. Just send a short message to [info.sonar@rero.ch](mailto:info.sonar@rero.ch) with the **name of the institution** and the **name and email address of the person in charge** (who will be assigned the [Admin role](https://sonar.test.rero.ch/help/user_roles/)).
```

Comment:
Ici je mettrais plutôt: "the interest the project is raising in our community" ou bien "So as the community can get acquainted with the future service, the SONAR team....to test it."